book_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'book'
+require 'rspec'
+
+describe Book do
+  let(:book) { Book.new('Test Book', 'Test Author') }
+
+  describe '#initialize' do
+    it 'creates a new book with title and author' do
```

Comment:
- [ ] Great job testing most of your class team. You already have great tests in place, But I notice there are some classes with major methods that have not been tested yet. Please test the methods inside the `App` class. This includes methods like `all_books, all_people, create_person, create_book, create_rental and some others`. Please keep in mind we need to test all the classes and all the methods inside those classes. 
- [ ] Also, Please check the namable class for raising an exception. We need to assert it is raising the `NotImplementedError` error.